docs/specs/Mountain State Adventure Destination/SPEC-07-adventures-hub-filtering/Research/How to build a locations database for tourism webs.md

```
@@ -0,0 +1,288 @@
+<img src="https://r2cdn.perplexity.ai/pplx-full-logo-primary-dark%402x.png" style="height:64px;margin-right:32px"/>
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)